stock_quant_package_dimension/models/stock_quant_package.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class StockQuantPackage(models.Model):
+    _inherit = "stock.quant.package"
+
+    pack_weight = fields.Float()
+    pack_length = fields.Integer(help="length")
+    width = fields.Integer("Pack Width", help="width")
+    height = fields.Integer("Pack Height", help="height")
+    volume = fields.Float(
+        "Pack Volume",
+        digits=(8, 4),
```

Comment:
```suggestion
        digits=(8, 4),
```

To be consistant with https://github.com/OCA/product-attribute/blob/18.0/product_logistics_uom/models/product_product.py#L17

Introduced in v16.0 https://github.com/OCA/product-attribute/pull/1314

Reply:
Hmmm... It will introduce more changes. It's a nice to have. Not blocking.